src/server/config.js

```diff
@@ -6,71 +6,30 @@ var fs = require('fs'),
 var config = {};
 var simulationFilePath;
 
+// module properties
+[
+    'liveReload',
+    'platform',
+    'platformRoot',
+    'forcePrepare',
+    'projectRoot',
+    'server',
+    'telemetry',
+    'simHostOptions',
+    'xhrProxy',
+    'touchEvents'
+].forEach(function (prop) {
```

Comment:
This is a good approach, but we need to add the [optional getter flag](https://github.com/Microsoft/cordova-simulate/pull/20/files/956f9d12da6cede37090e57d5a0b4f473b159213#diff-852bb59fffeac067fe0e2cbf7646bc73L103) as well in addition to the `single` setter flag.